### PR TITLE
Fix #72 - Add more Docker images and add port example

### LINKED ISSUE #72: Add comparing against some other images
- tobi312/rpi-mariadb
- [linuxserver/docker-mariadb](https://github.com/linuxserver/docker-mariadb) (looks like a serious project too)

### PR DESCRIPTION
**Pull request by williamdes**
Fixes: #72 
- Add more Docker images
- Add port example

### CODE REVIEW

**Review by jbergstroem — APPROVED** [review]

**Review by jbergstroem — CHANGES_REQUESTED** [review]
Looks like prettier isn't happy!

**Comment by williamdes** [comment]
> [warn] README.md
[warn] Code style issues found in the above file(s). Forgot to run Prettier?

So verbose 😄 


```
williamdes@williamdes:/mnt/Dev/@williamdes/mariadb-alpine$ ~/.yarn/bin/prettier -c .
Checking formatting...
All matched files use Prettier code style!
```

Okay, nobody wants to help 😄

File: README.md [hunk 1]
```diff
@@ -97,11 +108,12 @@ are unset unless provided.
 ### Adding your custom config
 
 You can add your custom `my.cnf` with various settings (be it for production or tuning InnoDB).
+You can also add other `.cnf` files in `/etc/my.cnf.d/` they will be loaded.
 Note: this will bypass `SKIP_INNODB` since it is injected into the default config on launch.
 
 ```console
 $ docker run -it --rm --name=db \
-         -v $(pwd)/config/my.cnf:/etc/my.cnf.d/my.cnf \
+         -v $(pwd)/config/my.cnf:/etc/my.cnf.d/my.cnf:ro \
```
Comment (williamdes) [inline README.md:116]: added read only security

**Comment by jbergstroem** [comment]
> So verbose 😄

Couldn't find a github action that is fast enough and gives you annotations. Guess I will just have to create one myself!

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -35,3 +35,9 @@ jobs:
         run: npm install -g prettier
       - name: run prettier
         run: prettier -c .
+      - name: run prettier
+        if: ${{ failure() }}
+        run: prettier -w .
+      - name: show prettier diff
+        if: ${{ failure() }}
+        run: git diff
```
Comment (williamdes) [inline .github/workflows/lint.yml:43]: Let me know if this is okay in this PR

Comment (jbergstroem) [reply .github/workflows/lint.yml:43]: Lets remove it as part of landing it; I want to annotate output from prettier.

Comment (williamdes) [reply .github/workflows/lint.yml]: Done, removed the commit

**Review by jbergstroem — APPROVED** [review]

**Pull request merged**